2017-08-21 13:02:25,655 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:44) (30690ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='212f67dca0491061ccd56ea38d1aabfa0b1cc46c0c688cb2befaf83a51f9e97b7abe3ecd7718313e3935ebcd8e9617036ea57c877cd1e34407311e36b962b311aa9cce07e42eab18c719e8e465aad59b61131828a5fbb73864ae0adc8d2fad560bed7183610f2e5f722901f4b5cb0dc262fc58d12cf3012e301e7eccf415fce2'}
2017-08-21 13:02:26,117 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:59) (31152ms)] - 【login.do result】###########################返回数据：{password=123, msg=请求成功, errorCode=0}
2017-08-21 13:06:58,026 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:46) (9497ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='1dd6e1aab90c68c0a9e3b2d42f3a35ab7f06afcdac63c277ab18cddad17c8f7171904b5f52cc7b204e077247b7affbecfc90da1b7496be865585128f0558befde2b8234e69404817527b3f252954b7dfa18e10c82e4292ee7a4807cd54eeeef85fc997583cb383d6ac5df1bd9b5cf322faedf6b33159aeeb9083b7d250043aaf'}
2017-08-21 13:06:58,454 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:71) (9925ms)] - 【login.do result】###########################返回数据：{password=ICy5YqxZB1uWSwcVLSNLcA==, msg=请求成功, errorCode=0}
2017-08-21 13:09:12,569 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:46) (144040ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='72b340c99a492c3f8a42612bd59dcb99b6a3387b0e17e123ea499161ca26f6567f491c5d0160f6a3138595c5f2191b46632391b3351d974f05f1fd752d9da1e47fd73d15250d86b2042fde96862fda5662594fd25a5422863bfc511129e7d4600a7a106c63106757104cb50d99b0e67d7ac3f3fb053ec738e217065f38028834'}
2017-08-21 13:09:27,197 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:71) (158668ms)] - 【login.do result】###########################返回数据：{password=ICy5YqxZB1uWSwcVLSNLcA==, msg=请求成功, errorCode=0}
2017-08-21 13:14:14,235 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:47) (9868ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='55dd59f57d39b29551b6f8f6998cb7bf567dd56c16882d55171780a44ddbeea5deb5a163a86b340b92cf8c44d956625a1066e6b1be329ae771ef8aa146cfae44a68149e0c9856b6201322bce6f94467a38c4215e03bf3fede5d9a1dc8b59bd6e49cf3b67654e95d90717121f319e4b45df4e04244e339ef8de02991c4a1e5358'}
2017-08-21 13:14:14,618 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:69) (10251ms)] - 【login.do result】###########################返回数据：{password=202cb962ac59075b964b07152d234b70, msg=请求成功, errorCode=0}
2017-08-21 13:26:40,441 -[ERROR]  method:[org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:331) (4427ms)] - Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.service.LoginService beacool.controller.LoginController.loginService; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginServiceImpl': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:538)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:229)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:298)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:725)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:403)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:306)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:106)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5003)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5517)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:901)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:877)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:652)
	at org.apache.catalina.startup.HostConfig.manageApp(HostConfig.java:1809)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.util.modeler.BaseModelMBean.invoke(BaseModelMBean.java:301)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:819)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at org.apache.catalina.mbeans.MBeanFactory.createStandardContext(MBeanFactory.java:618)
	at org.apache.catalina.mbeans.MBeanFactory.createStandardContext(MBeanFactory.java:565)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.util.modeler.BaseModelMBean.invoke(BaseModelMBean.java:301)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:819)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at javax.management.remote.rmi.RMIConnectionImpl.doOperation(RMIConnectionImpl.java:1487)
	at javax.management.remote.rmi.RMIConnectionImpl.access$300(RMIConnectionImpl.java:97)
	at javax.management.remote.rmi.RMIConnectionImpl$PrivilegedOperation.run(RMIConnectionImpl.java:1328)
	at javax.management.remote.rmi.RMIConnectionImpl.doPrivilegedOperation(RMIConnectionImpl.java:1420)
	at javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:848)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:322)
	at sun.rmi.transport.Transport$2.run(Transport.java:202)
	at sun.rmi.transport.Transport$2.run(Transport.java:199)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.Transport.serviceCall(Transport.java:198)
	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:567)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.access$400(TCPTransport.java:619)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler$1.run(TCPTransport.java:684)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler$1.run(TCPTransport.java:681)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:681)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.service.LoginService beacool.controller.LoginController.loginService; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginServiceImpl': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:555)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331)
	... 60 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginServiceImpl': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:538)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:229)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:298)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1081)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1006)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:904)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:527)
	... 62 more
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:555)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331)
	... 73 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1261)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1009)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:904)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:527)
	... 75 more
2017-08-21 13:29:28,871 -[ERROR]  method:[org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:331) (6036ms)] - Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.service.LoginService beacool.controller.LoginController.loginService; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginServiceImpl': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:538)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:229)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:298)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:725)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:403)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:306)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:106)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5003)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5517)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:901)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:877)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:652)
	at org.apache.catalina.startup.HostConfig.manageApp(HostConfig.java:1809)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.util.modeler.BaseModelMBean.invoke(BaseModelMBean.java:301)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:819)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at org.apache.catalina.mbeans.MBeanFactory.createStandardContext(MBeanFactory.java:618)
	at org.apache.catalina.mbeans.MBeanFactory.createStandardContext(MBeanFactory.java:565)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.util.modeler.BaseModelMBean.invoke(BaseModelMBean.java:301)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:819)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at javax.management.remote.rmi.RMIConnectionImpl.doOperation(RMIConnectionImpl.java:1487)
	at javax.management.remote.rmi.RMIConnectionImpl.access$300(RMIConnectionImpl.java:97)
	at javax.management.remote.rmi.RMIConnectionImpl$PrivilegedOperation.run(RMIConnectionImpl.java:1328)
	at javax.management.remote.rmi.RMIConnectionImpl.doPrivilegedOperation(RMIConnectionImpl.java:1420)
	at javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:848)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:322)
	at sun.rmi.transport.Transport$2.run(Transport.java:202)
	at sun.rmi.transport.Transport$2.run(Transport.java:199)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.Transport.serviceCall(Transport.java:198)
	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:567)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.access$400(TCPTransport.java:619)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler$1.run(TCPTransport.java:684)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler$1.run(TCPTransport.java:681)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:681)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.service.LoginService beacool.controller.LoginController.loginService; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginServiceImpl': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:555)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331)
	... 60 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'loginServiceImpl': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:538)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:229)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:298)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1081)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1006)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:904)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:527)
	... 62 more
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private beacool.util.MD5 beacool.service.impl.LoginServiceImpl.md5; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:555)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331)
	... 73 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [beacool.util.MD5] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1261)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1009)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:904)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:527)
	... 75 more
2017-08-21 13:30:30,932 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:42) (9550ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='073729f35713e3241c45dc66865c46f15edd40fcbe6d9d71052d91fc6aee03312d510781f7d6bb7fbb8f28b23034d3c7b800cd7da806922581d02e76b8730cb91f1eeb0572e8c6947e43c6e573057f4fa52af06fdbbb737ff24eff44ca66ce2a75bcae6d393ebdb5771f546c79195491a9fd5a335eb7ee28f604f8efbf856062'}
2017-08-21 13:30:31,410 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (10028ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=null, errorCode=0}
2017-08-21 13:33:25,050 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:42) (9155ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='0e91c352459fd964c92358245ef1721d85c7098fdb1747e76550f6a294b62134fe2183769b60d35c6223dc0bae1d5f107b27f8433b013f316930ada3a26217b604f405441700297987fc248208ade3b2f8ee1200ed37ef5fae5f46d092468672a2ddc0471143da4ac398d5d5be5b2af745b10f73f8fb172c2fde2279cb44017a'}
2017-08-21 13:33:25,654 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:78) (9759ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, data=null, errorCode=-1}
2017-08-21 13:37:49,861 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:40) (33537ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='8c5081981e7facadc0502071da7bb0326ce74f1cfa8cb466352e89cbd418ee33cdca91b6134ce81f8a3af1f61d4067c85fbd9a85943634559860066cbaadf3e8c2c8cd418f557862382b4a45965a111d437d5e04865bcde352a740240e76544468d62dc1a525141effc2d3c9b8c13e77f1b1dcbc65abd26c1816a041504ace94'}
2017-08-21 13:38:43,877 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (87553ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='7c2cc40a0f1d2f5a328526549044ccf6adc74d9445abe6cf57ec02bad00c213bdd3fb124aa2613874ab5f8092751acb772de9e1294b7a0d8fb58766c32443e980f57f200b21459b638c197d817df583c08df0ec8ceb52ffbd487e652c0b24e95f148e6bda22a7f2e480e2909b44a3bac8fc6bf52ba753e28ed95069c2d339724'}
2017-08-21 13:38:46,754 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:76) (90430ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, data=null, errorCode=-1}
2017-08-21 13:38:50,652 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (94328ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, data=null, errorCode=-1}
2017-08-21 13:39:40,972 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (144648ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='1aed2475431a1925dd86642dedbc1b4cb8aef44eefd61f2135267deedd5a96e81c453ca08f8362971426eb17e4832f67cad1c9cae6b56834ca3ce33d6c471e90d7865b33727a2f441f548171feac299c025ae5c65bb36dc9869798ff631e9abdb85469b67e8637e204e9e3adcd3bd6c985c0444c8747fcbfcfe2422c186dd5ed'}
2017-08-21 13:39:45,143 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (148819ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, data=null, errorCode=-1}
2017-08-21 13:40:52,947 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (9195ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='9fc3ee71ee61b25c2b6a9038a264e28ece9e8d0fc89d7c26bf0d6b10ea1ab7a0a0df4d90aac2a995649f198882d36f3793677d291a0533a4b039da027d0ed059c908cc0078431f98f3b9a647030744472bed5390fce71eef394ef80afb743510a15cf20e11487713056b0c152421d0d3780114a000ab2152acc1da32a927b758'}
2017-08-21 13:43:12,517 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (148765ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='1ac6b0e7c4c4287ca79f005c6e44cae4247331c6b50b99d915c8ed2c7844508fb1fc3363c065f4c1a362f899ad85360a3277ff26939beb99792c84f7a943acca5dbcfd065ef6e29ac8236b5995d7151ef70d0e16fc1e604f529ef6da58e40412714047db054ef0e05859bd6928c6768be5daceb07db63d3831bc26069a4663bb'}
2017-08-21 13:43:12,527 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (148775ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, data=null, errorCode=-1}
2017-08-21 13:43:12,562 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (148810ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, data=null, errorCode=-1}
2017-08-21 13:46:11,246 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (327494ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='3ed107dee705bbd2390625f717431a1402d53b39ebca2c94ec9aa78397fc86532e77a53ba447ca7cb40c953dfa0331604dc8b783b1fbf947cba29ad9ac8cb74bbb5b05631a2ef059bc9d1b8b0677a0830b09759e5f40b3508fde0d7b51a1fc411ac25fee4d095aeeee9f3e8bc051675766817ba74cccea7a55fa2bf234658c87'}
2017-08-21 13:46:11,326 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:70) (327574ms)] - 【Exception】========异常信息:java.lang.ClassCastException: beacool.entity.Login cannot be cast to java.util.Map
2017-08-21 13:46:11,326 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (327574ms)] - 【login.do result】###########################返回数据：{msg=数据库操作错误！, data={}, errorCode=1003}
2017-08-21 13:47:20,867 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (397115ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='39cb3bcf49b62432e7666ba83323dfcac04610145ff56d18141a8a999b6bf89cc482d196b249445e3c01b91290226bfb4577316244b04410d0f61dee2261f17bc86ff7c2d5a851d338e19d6315711e5e1e287486c2c58040d2d2cc04776ca3e134d2b84983cb3029cd9ee8e6e1a5837bd3fc7e199eb73fd14d3aa2901c911c56'}
2017-08-21 13:47:20,933 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:70) (397181ms)] - 【Exception】========异常信息:java.lang.ClassCastException: beacool.entity.Login cannot be cast to java.util.Map
2017-08-21 13:47:20,933 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (397181ms)] - 【login.do result】###########################返回数据：{msg=数据库操作错误！, data={}, errorCode=1003}
2017-08-21 13:47:33,689 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (409937ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='8fab8055bce33571e77cf16f79de79c5b5b77eaad59caeffedf6ab19aaa3b3d8183fd5ca55be527ab80ceb4a210e4ee9d4e1f16a694b729fc7aa80508b72b8318a546d58037f96b4ffb5d329d00e81f309838773305ed7d97e52d4a03daea95de4793b2aa95c6419a5df5c20fd074ec78506f2dd589a3673a595221f0551ec03'}
2017-08-21 13:48:11,245 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:70) (447493ms)] - 【Exception】========异常信息:java.lang.ClassCastException: beacool.entity.Login cannot be cast to java.util.Map
2017-08-21 13:48:11,245 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:75) (447493ms)] - 【login.do result】###########################返回数据：{msg=数据库操作错误！, data={}, errorCode=1003}
2017-08-21 13:49:04,407 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:39) (8798ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='3ce89818ecef3bbadbc646ef613ad170de4435c9513b94c25a72201b502f2ba67efca50a23fc2fcd002dc711a118040305336325d5283ebf46b0bdd8b00a8cddbe4e7ada0c0a21d2d8ae3f6601753250aff3062a31bf6b6a3f6c3514de3ee6b20b6c2fe7264391c17ccada7916e302441bcb078de2220fd77f086daddc30e41f'}
2017-08-21 13:53:52,995 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (9369ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='1583180b1bd277eafb50ac70675652102154f064ea5be93e6b8e60f91e72c32ab8b1348cfe8fce9f9ffaf93c4e22ec766a26eb4893999406d5991ac479550083a3295d494306b8dc59a804fd66aa10c5e20f8855cc3b0c413b97163c5f0351d3211286d91bb6666b4d0a4df6975942021c27eb6669e1c32b31937bb1ca64396c'}
2017-08-21 13:53:53,522 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:77) (9896ms)] - 【login.do result】###########################返回数据：{list=[Login{userName='admin', password='null'}], msg=请求成功, errorCode=0}
2017-08-21 13:55:58,814 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (12501ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='027a9f3b06627de4071e263eb21f9f653cdcf96b5743fb0ff3c8e7af77ce3710d034a98fa13fe01a18f1213c7ef41287d06100be26a5f2517eeeb492e4b6f5c05ec9bbfafa69ec206120de92fab61d8fd2263f1f3cc27edc0adc6972efe56b523d96e082997e8d619f7c11ba1bf1bc15422623e8a5cba4d006ec29a31f14d840', id='null'}
2017-08-21 13:55:59,342 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (13029ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1'}, errorCode=0}
2017-08-21 13:56:36,384 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (50071ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='0aaec4d14320537a78c7f04b5c543ecf5aef44a99d2ee3992bf7384da138596a064335474347d2da022e4802bcd408541c4af00a3b4be082940524edbb7a6870e0629c9c2b5dce9fed1e45a035a2e26c19cc66d2f200005c042b502878a6fedab6163057f0b8e36038e9eaa2b7778deea061340caf17aaa6d9aec6b6cdc438e7', id='null'}
2017-08-21 13:56:36,422 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (50109ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1'}, errorCode=0}
2017-08-21 13:56:57,701 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (71388ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='02529d1f143f673326a59a1e040c40cc79cc8c1b54ce9f119d4ce789664b82fb3ee8f273558efada6f3bd621dd253151a22ed8b383f93e7a18abc30b2ada0b9ab0f810c33b35364b130e7b9220fdf628dcc1046247c8a62dc653adf4164692e871d0ae1972f78713135aa41036187ea9993a0bbb79681c3abdcfd0f545062365', id='null'}
2017-08-21 13:56:57,816 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (71503ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1'}, errorCode=0}
2017-08-21 13:57:11,140 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (84827ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='0050f103e7d28f1446ff947629892d5c955a0b0544e81bb03d2ce16bd2da9c04863cc4f88c83863953ca5f0a9428b1e6436728feb57771911f3ee58ac9c818aabbfb8d2b92fb497663178315048bd3a69e38ec1ea2630074938c2a2988fbb04de3ad3a3a1728a4b84c3272785862c66a15382a615686b831cd84f4d0a22d257c', id='null'}
2017-08-21 13:57:11,173 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (84860ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, errorCode=-1}
2017-08-21 13:59:30,487 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (224174ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='55bc7a2c766a62bd656abd46704571b8c6837c45a8121301123825bcabd22f120740d35326fcfa93d2b4b91c74b2f9d1708e25ad83ae717428ae017a9bae9afbd6d655f62e036ca2450bfedeccb08f2f8283233f75b804c14c9ff3de108a974d49efd475b4fa4b55b13a84971f387b6d3b423dcf191514afa02cc8d8ad465784', id='null'}
2017-08-21 13:59:30,548 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (224235ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1'}, errorCode=0}
2017-08-21 13:59:39,028 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (232715ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='c45bb0707ebf84dacec12208956caf39ab4b700d59d25767a143d67c5f48ef8a3d086013199d984aa54ca1566e0d13ea9c32f859472351e68455777f78f623c84d3f80a5bb5e07a2aba3b2d7ce444ca75a037253abb5e034368bcc94c327d263333086deb23e756b512f2be4bc1700556037fb450f0803a7e3414ce981d30b30', id='null'}
2017-08-21 13:59:39,067 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (232754ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1'}, errorCode=0}
2017-08-21 14:00:20,319 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (274006ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='0b5b77f873f5bca1ae9b5cadfa02ca9ead088e8c83d8e03d3213682c61c59f55badd91c0b3162f09a17f64dd00c7e172b34d82c82a7c4cb8d4414c43662d901050726630c9d62d4fae09ba2ea8774bd1eb43823cf81481cbe217c4684b01f866e57c5b088570dac7028f285c2e6f6eff83c459a977fdc7dd7e7db36261cdead8', id='null'}
2017-08-21 14:00:20,353 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (274040ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, errorCode=-1}
2017-08-21 14:01:23,438 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (337125ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='79eb6abf8ec784cc86432e3de7375fc85182f0cf7093aeec28031b9af3adfa256f62fc839da678bbbf80c8fb6d345fb758076706efc6bf03591d56e05b82b2be19cc9bf80d3bcdb20468829f61f96c11a2191a049465e54cc930dfbc691f52e2b1b0b4bab1df41599b46aea1b6fb0f5175e270415f19be927c005b74fa31be39', id='null'}
2017-08-21 14:01:23,514 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (337201ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, errorCode=-1}
2017-08-21 14:01:40,772 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (354459ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='6c12483dbecbd9f5bc51d0e465c6db73e5b89c49e63c4788dff795ea0678b67fcdd1a4f337687e2792a32ac14db9f18ce0ae1b8d7842018814e75adcff2c505a4fe4748c303ba55ef64f711871168c69dabe41511765394f3f691f9294dec681f30a6d867f86b06373b46ca6c3cb413b724a01012d2a12b7320b0df834d93b9f', id='null'}
2017-08-21 14:01:40,806 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:79) (354493ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, errorCode=-1}
2017-08-21 14:29:31,998 -[ERROR]  method:[beacool.service.impl.PatientTempServiceImpl.queryPatientListResult(PatientTempServiceImpl.java:60) (2025685ms)] - 【findPatientList.do params】############################入参信息:PatientTempParam{sex=0, idCard='null', name='null', position='null', tempDic=0, hospitalization=0}
2017-08-21 14:30:02,252 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2055939ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$1.lookupAllHostAddr(InetAddress.java:901)
	at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1295)
	at java.net.InetAddress.getAllByName0(InetAddress.java:1248)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,265 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2055952ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,270 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2055957ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,294 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2055981ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,296 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2055983ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,321 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056008ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,322 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056009ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,337 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056024ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,338 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056025ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,353 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056040ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,354 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056041ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,370 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056057ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,371 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056058ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,386 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056073ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,388 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056075ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 18 more
2017-08-21 14:30:02,402 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056089ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,403 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056090ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,419 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056106ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,420 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056107ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,435 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056122ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,436 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056123ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,448 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056135ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,449 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056136ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,467 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056154ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,468 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056155ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,484 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056171ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,486 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056173ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,503 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056190ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,504 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056191ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,523 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056210ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:02,524 -[ERROR]  method:[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1651) (2056211ms)] - create connection error
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor103.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1036)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2232)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2265)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2064)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:790)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor104.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:377)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:395)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:211)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1360)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1414)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1649)
Caused by: java.net.UnknownHostException: mysql.rdsm4caepnbupc3.rds.bj.baidubce.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1252)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:190)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 17 more
2017-08-21 14:30:32,988 -[ERROR]  method:[beacool.service.impl.PatientTempServiceImpl.queryPatientListResult(PatientTempServiceImpl.java:79) (2086675ms)] - 【result】###########################返回数据：{list=[PatientTemp{temp=39.2, position='207室', hospitalization=1}, PatientTemp{temp=37.5, position='208室', hospitalization=1}, PatientTemp{temp=37.2, position='201室', hospitalization=1}], total=3, message=请求成功, errorCode=0, totalPages=1}
2017-08-21 14:30:42,140 -[ERROR]  method:[beacool.service.impl.PatientTempServiceImpl.queryPatientListResult(PatientTempServiceImpl.java:60) (2095827ms)] - 【findPatientList.do params】############################入参信息:PatientTempParam{sex=0, idCard='null', name='null', position='null', tempDic=0, hospitalization=0}
2017-08-21 14:30:42,205 -[ERROR]  method:[beacool.service.impl.PatientTempServiceImpl.queryPatientListResult(PatientTempServiceImpl.java:79) (2095892ms)] - 【result】###########################返回数据：{list=[PatientTemp{temp=39.2, position='207室', hospitalization=1}, PatientTemp{temp=37.5, position='208室', hospitalization=1}, PatientTemp{temp=37.2, position='201室', hospitalization=1}], total=3, message=请求成功, errorCode=0, totalPages=1}
2017-08-21 14:43:54,607 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (9582ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='3c58b341fba6bc1ccaac793717b1222f059413f7049b2561bc4e1f6deae30e1998fda1af83a6272a1237256f26752a0a0abd3be7ac4c8c829a3e2e1db4f143d08b66899f73757a795f4215d408239ef9d66df80a1f2bf9fb53585d471aae5fe5403f65021b3c115a158b26edac9db69db61eb2b690f2b3eced78a32215eeeecc', id='null', token='null'}
2017-08-21 14:43:55,495 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:82) (10470ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='null'}, errorCode=0}
2017-08-21 14:45:08,220 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (8309ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='59c8036afbb5a9e618087299c93c65fc7eb2e6a5a2032ee80bf550728006b78f757b239c31b4ade32a9edf69434ac8dd6249cd25428857ee86786f8f34647885fd30a6b0f591fc7a03b5ae9d7f03b1b20ffdf184f72e8676e6f70722546f5f46e76274af3c745fc0c7f59093012f6f3ffeec4b1f808a103e379b7859e1fcae16', id='null', token='null'}
2017-08-21 14:45:08,884 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:84) (8973ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='null'}, errorCode=0}
2017-08-21 14:46:49,373 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (56890ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='5493185f212475ba29e47e016b0a1651c0f28f44cec0ad8fdd84a9a9d2cc3dec444b2c2e649fa3eaa1befed3f198ad695f3e58dac8365422f075bb6e186054ccb2161436ea52af4d77778777f3c5a0da929d093b8d8432c5d6aaf7d8224481d0ba43dcd6c2a29b7e538d4a58fb80e855f90da62060e09f29a8107dc43c5f0d43', id='null', token='null'}
2017-08-21 14:49:27,043 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (214560ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='null'}, errorCode=0}
2017-08-21 14:49:54,072 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (241589ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='924a90d85128516caa920508329d0ccff3f17b3e9657b570ab6c81e446f722726bbc1fdecbac0409851f384f9688219df0fab97c571a46347514f66128e9286c429d21ae5c0403690da905cc8a0df57135f59645bcb0696fd07d53341918bf39aea6c706f07e21383b4c681df74f83edc46a4a8cbfe9d0aa71f3bd099dfd5865', id='null', token='null'}
2017-08-21 14:50:06,569 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (254086ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='null'}, errorCode=0}
2017-08-21 14:50:58,721 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (10075ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='1a3cb94557dba117d7583e89db6e5f17c69382c7c7ccc109f3b23405d70342c6f35a6fefb3f5bbe2ded13f8ed323bd1a982153cab79617c36d0800130a1c0ce7e1178745af281e0691b97ce1d6a86247a7d9578e026ec32ff5fa60fd474bc21b583f82549517b9e4ed9dcd0e7d50a1484ed335dd4fc918ef6e3e3c13442e7e25', id='null', token='null'}
2017-08-21 14:50:59,409 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (10763ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='a82f6f3070054d3ea8a3a58ca32163f5'}, errorCode=0}
2017-08-21 14:52:50,961 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (122315ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='8c3e8a00954593589ad84dec142ab75b78d9f2b14ec22441d39f51ddd8d12a3738e8645f1744df97c7f8542a8f54366fd6107a977ac05d3f1bae084b7edd8626a1a343cad77b3412ec329c287bd29f4ee5f8410f1a09ef2ca784b99003f6c4c1758e8b6866e6ad474540bcf7fd0859203ee9cb244b18a18a4b621b5c0a75beff', id='null', token='null'}
2017-08-21 14:52:51,163 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (122517ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='2ce6083d490d4517804f20d9de1f6ebf'}, errorCode=0}
2017-08-21 14:55:47,468 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (298822ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='465bc7de69f46a9fba348f13a36a664119cac55e8d706964a940c98bb84a4c04f330c7ec27c0b785ac97a03db810a57f363f8833ece7275f18ffd46f43b06ccbb41979790788979f7fbdccedb94d5f4374ab49e241f3e064bece2b750e5ee68c6b8e7251639a44dfb467e50031fe93ffaa0ba7ea8e3dd568cc243f24198d53b6', id='null', token='null'}
2017-08-21 14:55:47,669 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (299023ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='dfa39789500c4276995c218bdb160b2b'}, errorCode=0}
2017-08-21 14:59:09,327 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (500681ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='1de7615d46c4bede22a417d742e15fc5b8d2f5e2ea3670619c400cf546a1bdc9c5cf7be04ea7029aac58d0189235b0fbb66668842469f4f52f4012ad4f6ed8752b58d7d27ad6240c1aab9db82bffea33a7f16aee5f2749858f70823ffb7efa9957fa397ddcc65947abcd6e470fd8f819495efd1d14a11f22eff301daf9c92d9d', id='null', token='null'}
2017-08-21 14:59:09,522 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (500876ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='ba3b72560e59478b88ed35626bf4598d'}, errorCode=0}
2017-08-21 15:03:32,877 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (9455ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='5454341451f52113044560c2b8dd67709ba09ce35cc5719b14ae38e1bb38718b44e033de462e653a013ce61d809ebcc9bc53901fb0b560a08cc49083f67ab5a07f8620fca66b7dad45fc5bb7f489b77e836a46b61c60aea9be469df9e771e997ac47eabbe36bc8b77e26167c604457d9524f5e26e25f887b59ac60d0d4d5593e', id='null', token='null'}
2017-08-21 15:03:33,516 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (10094ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='d75602dc219d405e9c39bfdcaa9db372'}, errorCode=0}
2017-08-21 15:04:14,273 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (50851ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='bebcc259930f77fe4a2ca0e84e7ae1e285ef6571a971fac57ad5e4416e0bc5e5f22efb812b5284fe2b311f1b5a417fbc6a1ce64fd8a7e33b9f4883f6220ed25c5000b5593b528138abf8c6d78d4019cecad5172cd21dbe688958879360cb6c6b8f1df226c4d84d538966bcae51bade0282e8edf22c050dbe88fff42c52dd5cfd', id='null', token='null'}
2017-08-21 15:04:14,311 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (50889ms)] - 【login.do result】###########################返回数据：{msg=账号或密码错误！, errorCode=-1}
2017-08-21 15:04:32,508 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (69086ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='7e2acf32b10deb643bbb281565e1032c8ab7b260764d624aa0ff912496e1193a466ab6ee3cc55bacdceaf393839f20facfbf5687815ac04f146fff9ab78e70b90fda7747b83f313955ec337a72137b78ed0de1917b69212acf49ebbfc84b073cddcf137780ad6f2b9a34952a040a2468d9d6ad08041a05c41c6809f521f37e78', id='null', token='null'}
2017-08-21 15:04:32,616 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (69194ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='eabc44eb90ac4462b48a380b7663b169'}, errorCode=0}
2017-08-21 15:05:48,677 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (145255ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='236f5dd719ede07e1b62506b5179ca98259d92661192484167fe2e4bc69af84e569992b150e41a54104e385e48a1ce82f77d8218d374c95989ea0db077e4f2c7f3ece391f408a89c1151b5a2815870d73e2ea92e0c20871fa5dd7bc604092b2a640c350ac83695c40dedc5faf31f68357cab63ae53d4923a61f68d3cc99c9555', id='null', token='null'}
2017-08-21 15:05:48,775 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (145353ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='a18387187f8e4dce937fe31e62133fcb'}, errorCode=0}
2017-08-21 15:06:04,441 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (161019ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='55483df5b3186ad8ac74ab409baa6093cfc7c0738203b975f6f7b19d6826c6bf524b9d4b9396cf95245b3bf12f8de2dd4da9b7493c1c02ae370360f6707a2fb17c8cbf33098a2f07e85a2f95a576e569a7860a273c3b5973fc457f62136ab061d62521467b08aedac0f45db26ad92fef880bf57578bd68a3a01db7deef2901c7', id='null', token='null'}
2017-08-21 15:06:04,537 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (161115ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='3ff60cf8800c446ab8329a5945281cd0'}, errorCode=0}
2017-08-21 15:08:48,462 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (325040ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='137527f4fbf4913eea2f8582338983308ac1b7d4e45971b10bed0bce6bad1e096747876240e79d631c39fc80aadbfdbae0e67052f4e83fe43c3c29cf64211e4e05e8cdbc8a358eb52ce283227652c6d00008c9ced72431e52492d4bab98765e5be3bba33e5b202301b5cb5b8726c90e2e0f27581a14cc74e109aa4f1ca95c638', id='null', token='null'}
2017-08-21 15:08:48,568 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (325146ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='8de1b152c71a4a4bbfa8a86ffc266463'}, errorCode=0}
2017-08-21 15:09:39,576 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (376154ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='932e1dcf5b32ec79d4c3d50a5857acf1cbf68b60e9db9a001bdd91bc593a7e835c94fe38379607d445f715ddf66e28383edbaf152c186acbfca47763c2dd60e29cbbe52acc0455c76f02d3ab1922a3672a4006192b9d8f64be595408e1073e234a0b2fe4fabc25f565118d2fffba48ef7b699c7d695285712c2bf8217e223b11', id='null', token='null'}
2017-08-21 15:09:39,676 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (376254ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='3b7706c478014c409f6bc99504c2e84d'}, errorCode=0}
2017-08-21 15:18:44,908 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (12249ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='3662ccc7b2c729be00e83013258ab5adf50ff860429fe0773d6dff3c6f00d1ba6ab94ecfe7d3ba90cf9f0098466354e9dcb17b906318b0317d34eab876ae62fd4930652d588c6a3001c773824562fd5e78cab2700ce108d1e6d164a1243e8ea529e561023eec39e12583e2c0455e87aa0c6dc66e61e6edc16b88069cd894bd93', id='null', token='null'}
2017-08-21 15:18:45,548 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (12889ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='87db298702cd4397bd9862bdc10da2bd'}, errorCode=0}
2017-08-21 15:18:52,628 -[ERROR]  method:[beacool.service.impl.PatientTempServiceImpl.queryPatientListResult(PatientTempServiceImpl.java:60) (19969ms)] - 【findPatientList.do params】############################入参信息:PatientTempParam{sex=0, idCard='null', name='null', position='null', tempDic=0, hospitalization=0}
2017-08-21 15:18:52,738 -[ERROR]  method:[beacool.service.impl.PatientTempServiceImpl.queryPatientListResult(PatientTempServiceImpl.java:79) (20079ms)] - 【result】###########################返回数据：{list=[PatientTemp{temp=39.2, position='207室', hospitalization=1}, PatientTemp{temp=37.5, position='208室', hospitalization=1}, PatientTemp{temp=37.2, position='201室', hospitalization=1}], total=3, message=请求成功, errorCode=0, totalPages=1}
2017-08-21 15:45:52,663 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:41) (1640004ms)] - 【login.do params】############################入参信息:Login{userName='admin', password='16d6e0a6e585d23c98c5db6b21b5beffeb3d6c78bf9636eab788d135048d961725030ffd192999f3192d1f40b3bd7c890e190d965b6721f4f47e5ac95d656ff6e8da65c6055005a60430d51e4d43bfc92f63681f8ae5496773ae3a2e7fb9365ecf271577c1b236b9f3aaecef3ebffa948bfd2fe03a96fd9fcc120320975ef5e8', id='null', token='null'}
2017-08-21 15:46:19,083 -[ERROR]  method:[beacool.service.impl.LoginServiceImpl.queryLogin(LoginServiceImpl.java:83) (1666424ms)] - 【login.do result】###########################返回数据：{msg=请求成功, data=Login{userName='admin', password='null', id='1', token='dee21397c9664b06b94cd153db49aa24'}, errorCode=0}
